### DISCUSSION TITLE
今天全网的算力暴增

### DISCUSSION THREAD

**Discussion by love-chia** [Chia-Network/chia-blockchain#2799] [Ideas]
今天的全网算力暴增，基本上快翻倍了！看来应该是有大型矿池开始运作了。对于一般家庭个人的旷工来说，我们的这点算力又极大的被稀释了，获得XHC的概率又小了。现在看来如果不再进行更多的投资，就只能自娱自乐了！

**Comment by cyr616** [comment]
增长只会越来越快，明后天要突破1eb，早加矿池早有收益

**Comment by mustangup** [comment]
对抗矿池最好的方法，就是更多的人参与，稀释他的比例。
